### PR TITLE
Upgrade to babel 7

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -123,5 +123,8 @@
   "collective": {
     "type": "opencollective",
     "url": "https://opencollective.com/polished"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.0.0"
```
Comment (bhough) [inline package.json]: Is there a reason this is in dependencies vs devDependencies?

Comment (TrySound) [reply package.json]: This dep is generated in build step. Take a look at dist files

**Comment by TrySound** [comment]
@bhough Hey, no! This is not how babel runtime works. If user doesn't have babel runtime it will break their build!

**Comment by TrySound** [comment]
no!!!!!!!!

**Comment by bhough** [comment]
@TrySound it is a bot not Max. We are testing something, please give us a few minutes.

**Comment by TrySound** [comment]
Make prerelease please. Don't publish broken package.

**Pull request merged**